.drone.yml

```diff
@@ -85,6 +85,24 @@ pipeline:
       matrix:
         TEST_SUITE: phpunit
 
+  phan:
```

Comment:
there is a ``php-phan`` section in ``services`` - please investigate, probably delete it

Reply:
Removed.